src/main/groovy/au/org/ala/ecodata/reporting/ManagementUnitXlsExporter.groovy

```diff
@@ -19,16 +19,18 @@ class ManagementUnitXlsExporter extends TabbedExporter {
     List<String> activityHeaders = ['Activity Type','Activity Description','Activity Progress', 'Activity Last Updated' ]
     List<String> activityProperties =  ['type','description','progress', 'lastUpdated']
     List<String> commonActivityHeaders =  ["Management Unit ID",'Management Unit Name', 'Report ID', 'Report name', 'Report Description', 'From Date', 'To Date', 'Financial Year', 'Current Report Status', 'Date of status change', 'Changed by'] + activityHeaders
+    List<String> commonActivityHeadersSummary =  ["Management Unit ID",'Management Unit Name', 'Report ID', 'Report name', 'Report Description', 'From Date', 'To Date', 'Financial Year', 'Current Report Status', 'Date of status change', 'Changed by']
     List<String> commonActivityProperties = ["managementUnitId",'managementUnitName', REPORT_PREFIX+'reportId', REPORT_PREFIX+'reportName', REPORT_PREFIX+'reportDescription', REPORT_PREFIX+'fromDate', REPORT_PREFIX+'toDate', REPORT_PREFIX+'financialYear', REPORT_PREFIX+'reportStatus', REPORT_PREFIX+'dateChanged', REPORT_PREFIX+'changedBy'] +
             activityProperties.collect {
```

Comment:
You should build the commonActivityProperties / commonActivityHeaders using commonActivityHeadersSummary + activityHeaders instead of duplicating the lists?  Unless you've made a subtle change I didn't notice?